enos/enos.py

```diff
@@ -219,9 +219,13 @@ def install_os(env=None, **kwargs):
     else:
         kolla_cmd.append('deploy')
 
+    # NOTE(dpertin): kolla-ansible is not able to parse the --extra_vars
+    # parameter when vars are provided as a dict which contains spaces, so they
+    # are removed here
     kolla_cmd.extend(["-i", "%s/multinode" % env['resultdir'],
                       "--passwords", "%s/passwords.yml" % env['resultdir'],
-                      "--configdir", "%s" % env['resultdir']])
+                      "--configdir", "%s" % env['resultdir'],
+                      "--extra", "%s" % str(env['config']).replace(" ","")])
```

Comment:
we could add the specific vars in `generated_kolla_vars` and remove the need for the `--extra` option